roles/cobbler/templates/snippets/cephlab_rc_local

```diff
@@ -9,13 +9,45 @@
 #set script = '/etc/rc.local'
 #end if
 
-cat > $script << EOF
+cat > $script <<\EOF
```

Comment:
What's this?

Reply:
Without the backslash, $(these) are interpreted and run as commands.  https://stackoverflow.com/a/17983608